docs/api-reference/clusternetworkconnect-api-spec.md

```
@@ -0,0 +1,157 @@
+
+
+ClusterNetworkConnectSpec defines the desired state of ClusterNetworkConnect.
+
+
+
+_Appears in:_
+- [ClusterNetworkConnect](#clusternetworkconnect)
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `networkSelectors` _[NetworkSelectors](#networkselectors)_ | networkSelectors selects the networks to be connected together.<br />This can match User Defined Networks (UDNs) and/or Cluster User Defined Networks (CUDNs).<br />Only ClusterUserDefinedNetworkSelector and PrimaryUserDefinedNetworkSelector can be selected. |  | Required: \{\} <br /> |
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)